### ISSUE TITLE
Can we lock the "Back" button on the file upload page?

### ISSUE THREAD

**Issue by jmaferreira** [keeps/roda#942]

**Comment by nunovieira220** [comment]
The button now blocks itself and a message appears on the UI since 410ab7660099347f6c4430a8281df36e0b1bf67f.

This is how the message looks like (is it okay @jmaferreira?):
![screenshot from 2017-05-16 14 59 13](https://cloud.githubusercontent.com/assets/8982932/26110372/0c7aad04-3a4a-11e7-8968-1be38551ddce.png)

**Comment by jmaferreira** [comment]
Yes. Its perfect . 'm just not sure if the message should show up on top on bellow the rectangular thingy. Decide together with the majestic team: @hsilva-keep @chalkos @nunovieira220 

PS: Watch out the translation.
